client/CPlayerInterface.h

```diff
@@ -276,10 +277,10 @@ class CPlayerInterface : public CGameInterface, public IUpdateable
 
 	bool duringMovement;
 	bool ignoreEvents;
+	std::atomic<size_t> numOfMovedArts;

```

Comment:
Can it actually be accessed from different threads? PlayerInterface always accessed while in lock (`pim`)

Reply:
Just checked it works without atomic. Removed.